platform-api-client-gwt/che-core-client-gwt-account/pom.xml

```diff
@@ -102,6 +107,7 @@
                 <configuration>
                     <dtoPackages>
                         <package>org.eclipse.che.api.account.shared.dto</package>
+                        <package>org.eclipse.che.api.workspace.shared.dto</package>
```

Comment:
If you mean workspace *dto*, it doesn't work without it. 
`Account` dependes on `UsersWorkspace` it  means  that `AccountDescriptor` needs `UsersWorkspaceDto` which is defined in `org.eclipse.che.api.workspace.shared.dto`.
It would not compile if you remove this definition. 

Do you have an another solution?